inc/builder/type/header/mode-switcher/assets/js/unminified/customizer-preview.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * This file adds some LIVE to the Customizer live preview. To leverage
+ * this, set your custom settings to 'postMessage' and then add your handling
+ * here. Your javascript should grab settings from customizer controls, and
+ * then make any necessary changes to the page using jQuery.
+ *
+ * @package Astra
+ * @since x.x.x
+ */
+
+( function( $ ) {
+
+	var tablet_break_point    = astraBuilderPreview.tablet_break_point || 768,
+		mobile_break_point    = astraBuilderPreview.mobile_break_point || 544,
+		section = 'section-mode-switcher',
+		selector = '.ast-header-mode-switcher .ast-header-mode-switcher-wrap';
```

Comment:
```suggestion
		selector = '.ast-header-mode-switcher-wrap';
```
@imnavanath Can we reduce the CSS selector?

Reply:
Updated 👍 Now used this line only - `selector = '.ast-header-mode-switcher';`